README.md

```diff
@@ -39,14 +39,17 @@ test('Test the resolver', () => {
     },
   };
 
-  // parser.resolve() automatically typecasts (note JSON becomes a JS object for ease of testing)
   const response = parser.resolve(context);
 
-  // For convenience, the response is returned as a JS object rather than JSON
   expect(response.id).toBe('testId');
 });
 ```
 
+The vtl template
+```
+$util.toJson($ctx.result)
+```
+
```

Comment:
This template wouldn't work with the above example - if one is added it should include `id` so that the test succeeds.